### ISSUE TITLE
Doesn't work on sitecore 9

### ISSUE THREAD

**Issue by negkashk** [thecadams/301RedirectModule#47]
Hi,

I installed this module on sitecore 9 successfully with no errors. However, it doesn't work. I tried Redirect Url and Redirect Pattern Sample, and none of the worked.

I was wondering i you could let me know what I am missing.

Thanks

**Comment by negkashk** [comment]
Thank you!

**Comment by thecadams** [comment]
HI,
I know it's been a long time on this one, but I haven't been able to secure a valid Sitecore license file.

There may be others who can help - let me cc in some other contributors. cc @maxslabyak @Chris-Castle @mawiseman @ThomasBaek @timgriff84 - but of course it's all volunteer efforts here.

Want to apologise again.

**Comment by mawiseman** [comment]
We are currently using this in Sitecore 9 projects.
I'd recommend cloning the repo, and debugging locally.
If there is a scenario that is failing, fork, test and create a pull request

**Comment by skapadiaglobus** [comment]
Hi, 
I am using Sitecore 9.1
I have tried the following redirect pattern

I have a page like http://monograms.gfob9.com/deals
I want to redirect to http://monograms.gfob9.com/why-monograms/about-monograms
I am using this 

![Capture](https://user-images.githubusercontent.com/61958239/76182874-91064380-621a-11ea-9e83-60c21343ddba.PNG)





 


Currently local website runs on IIS, are there any IIS settings to be configured or any code or more config changes required to accomplish this.

If you could suggest.


Thanks
Shefali

**Comment by mawiseman** [comment]
I'd recommend you download and debug to see what's happening
your regex looks right

this module could probably do with a nice and detailed logging switch that lists what is being hit and where

**Comment by thecadams** [comment]
@mawiseman agreed on the debugging switch - I would add something if I had a Sitecore license to test with!
